bitcoind-rpc/src/main/scala/org/bitcoins/rpc/jsonmodels/RpcPsbtResult.scala

```diff
@@ -69,3 +70,25 @@ final case class WalletCreateFundedPsbtResult(
     fee: Bitcoins,
     changepos: Int
 ) extends RpcPsbtResult
+
+
+final case class AnalyzePsbtResult(
+                                  inputs: Vector[AnalyzePsbtInput],
+                                  estimatedvsize: Option[Double],
+                                  estimatedfeerate: Option[Double],
+                                  fee: Option[Bitcoins],
+                                  next: String
+
+                                  )extends RpcPsbtResult
+final case class AnalyzePsbtInput(
+                                   hasutxo: Boolean,
+                                   isfinal: Boolean,
+                                   missing: Option[AnalyzePsbtMissing],
+                                   next: Option[String]
+                                 )extends RpcPsbtResult
+final case class AnalyzePsbtMissing(
```

Comment:
nit: I would personally call this something like `PsbtMissingData` or something like this